Add table `aws_codedeploy_deployment_config` closes #1653 (#1662)
turbot/steampipe-plugin-aws: docs/tables/aws_codedeploy_deployment_config.md

# Table: aws_codedeploy_app

A deployment configuration is a set of rules and success and failure conditions used by CodeDeploy during a deployment. These rules and conditions are different, depending on whether you deploy to an EC2/On-Premises compute platform, AWS Lambda compute platform, or Amazon ECS compute platform.

## Examples

### Basic info

```sql
select
  arn,
  deployment_config_id,
  deployment_config_name,
  compute_platform,
  create_time,
  region
from
  aws_codedeploy_deployment_config;
```

> ### Get total configurations deployed on each platform

Reviewer: ```suggestion
### Get the total configurations for each compute platform
```

```sql
select
  count(arn) as configuration_count,
  compute_platform
from
  aws_codedeploy_deployment_config
group by
  compute_platform;
```

> ### List the user defined configurations

Reviewer: ```suggestion
### List the user managed deployment configurations
```

> # Table: aws_codedeploy_app

Reviewer 2: ```suggestion
# Table: aws_codedeploy_deployment_config
```

A deployment configuration is a set of rules and success and failure conditions used by CodeDeploy during a deployment. These rules and conditions are different, depending on whether you deploy to an EC2/On-Premises compute platform, AWS Lambda compute platform, or Amazon ECS compute platform.

## Examples

### Basic info

```sql
select
  arn,
  deployment_config_id,
  deployment_config_name,
  compute_platform,
  create_time,
  region
from
  aws_codedeploy_deployment_config;
```

> ### Get the total configurations for each compute platform

Reviewer 2: ```suggestion
### Get the configuration count for each compute platform
```
